<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Derivacion2</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valor de h:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valorH">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Puntos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="puntos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tres_puntos_boton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método tres puntos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cinco_puntos_boton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Método cinco puntos</string>
   </property>
  </widget>
  <widget class="QLabel" name="ver_resultados">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
